src/transformers/modeling_bert.py

```diff
@@ -976,13 +981,17 @@ def forward(
     """Bert Model with a `language modeling` head on top for CLM fine-tuning. """, BERT_START_DOCSTRING
 )
 class BertLMHeadModel(BertPreTrainedModel):
+
+    authorized_unexpected_keys = [r"pooler"]
+    authorized_missing_keys = [r"position_ids", r"predictions.decoder.bias"]
```

Comment:
Only this one and the `ForMaskedLM` should have a `authorized_missing_keys` for the special value.